src/registrar/utility/csv_export.py

```diff
@@ -273,6 +145,94 @@ def parse_row(cls, columns, model):
         pass
 
 
+class MemberExport(BaseExport):
+    """CSV export for the MembersTable. The members table combines the content
+    of three tables: PortfolioInvitation, UserPortfolioPermission, and DomainInvitation."""
+
+    @classmethod
+    def model(self):
+        """
+        No model is defined for the member report as it is a combination of multiple fields.
+        This is a special edge case, but the base report requires this to be defined.
+        """
+        return None
+
+    @classmethod
+    def get_model_annotation_dict(cls, request=None, **kwargs):
+        """Combines the permissions and invitation model annotations for
```

Comment:
You should have the return value as a comment here. I can tell looking at the code it's going to return a dict, but determining the contents of the dict take a minute to parse. Complicated/none standard return values really warrant a comment (Though the best practice is that you should always have return values/types in your function comments)